server/webapp/WEB-INF/rails.new/app/assets/stylesheets/vm/structure_for_old_ui.scss

```diff
@@ -945,9 +945,7 @@ a.xlink_as_button {
 .sub_tabs_container {
     background: #999 image_url('g9/backgrounds/overlay_bottom_inside_shadow.png') bottom repeat-x !important;
     border: 0 !important;
-    padding: 10px;
-    padding-bottom: 0;
-    margin-bottom: 10px !important;
```

Comment:
With this margin-bottom removed, the other tabs look a little weird(er), don't you think?

### Old:

![old1](https://cloud.githubusercontent.com/assets/172235/7915024/d678199c-0848-11e5-9a7f-492d84ba96cb.png)
![old2](https://cloud.githubusercontent.com/assets/172235/7915025/d67994de-0848-11e5-8630-867505ca9e9c.png)

### New:

![new1](https://cloud.githubusercontent.com/assets/172235/7915029/ddc18de6-0848-11e5-9686-4df2b84aef6f.png)
![new2](https://cloud.githubusercontent.com/assets/172235/7915030/ddc24f56-0848-11e5-8d9e-c07ed90541b1.png)